File: leetcode/Easy/26_remove_duplicates_from_sorted_array.py
def removeDuplicates(nums) -> int:
    temp = []
    for i in nums:
        if i not in temp: temp.append(i)

    return len(temp)

def removeD(nums) -> int:
    for i in nums:
        try:
            if i == nums[nums.index(i) + 1]: nums.pop(nums.index(i))
        except IndexError:
            pass
    return len(nums)
    
nums = [0,0,1,1,1,2,2,3,3,4]
print(removeD(nums))
print(removeDuplicates(nums))

File: leetcode/Easy/rust/26_remove_duplicates_from_sorted_array.rs
fn remove_duplicates(nums: &mut Vec<i32>) {
    let mut temp = Vec::new();
    for i in nums.iter() {
        if !temp.contains(i) {
            temp.push(*i);
        }
    }

    println!("{:?}", temp);
}

fn main() {
    let mut nums = vec![0, 0, 1, 1, 2, 2, 3, 3, 4, 4, 5];
    remove_duplicates(&mut nums);

}